aces/Loris
# [conflict_resolver] Rename TableName to TestName

## Brief summary of changes


#### Testing instructions (if applicable)

1. Try loading conflict resolver including an instrument that has a different test name / table name
2. Try resolving conflicts
3. Run the recreate_conflicts.php script, make sure there are no failures and conflicts are properly created
4. Check that the dashboard properly displays the number of conflicts
5. Test the conflicts tab of the Behavioural QC module to make sure it loads properly & links work
6. Double check that there are no other places in the codebase where the change should be made

#### Link(s) to related issue(s)

* Resolves #9242, #9248
